### DISCUSSION TITLE
How to disable the clock in lualine

### DISCUSSION THREAD

**Discussion by meleu-cw** [LazyVim/LazyVim#2025] [Recipes]
I don't like the clock on the right side of my lualine...

Reasons:
- I already have the time in other visible places of my system (systray and tmux line)
- when the time updates (each minute) tmux notifies me that there was an update on the nvim tab, which is distracting

Here's how I disable it:

Create a `lua/plugins/lualine.lua` with this:
```lua
return {
  "nvim-lualine/lualine.nvim",
  opts = {
    sections = {
      lualine_z = {},
    },
  },
}
```

If there's shorter way to achieve that, please share with me! 🙏

**Comment by leonasdev** [comment]
> Create a `lua/plugins/lualine.lua` with this:
> 
> ```lua
> return {
>   "nvim-lualine/lualine.nvim",
>   opts = {
>     sections = {
>       lualine_z = {},
>     },
>   },
> }
> ```

I think this was the shortest way to achieve this.

**Comment by ModemNakata** [comment]
finally I found the way! thank you
